.gitpod.yml

```diff
@@ -0,0 +1,12 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: npm install && npm run build
+    command: npm run start
+  - deploy: next build
```

Comment:
This whole file is a basic initial .gitpod.yml, and I added line #8 ~ 9. Please check my code. Thanks!